src/uct/ib/base/ib_iface.c

```diff
@@ -1050,7 +1052,8 @@ static void uct_ib_iface_set_num_paths(uct_ib_iface_t *iface,
 
         if ((iface->num_paths == 1) &&
             (uct_ib_iface_port_attr(iface)->active_speed == UCT_IB_SPEED_NDR)) {
-            iface->num_paths = 2;
+            iface->num_paths     = 2;
+            iface->bw_path_ratio = 0.9;
```

Comment:
can we calculate it in uct_ib_iface_estimate_perf instead of saving it on the iface?